SpeechRecognizerButton/SFButton.swift

```diff
@@ -212,6 +215,16 @@ open class SFButton: UIButton {
         }
         let normalizedValue = pow(10, averagePower / 20)
         waveformView?.updateWithLevel(CGFloat(normalizedValue))
+        
+        if(averagePower < -10){
+            counter += 1
+            if(counter / 60 >= 2){
+                endRecord()
```

Comment:
Can you add a unit test that covers this code please?

Reply:
I have added the spaces and changed the style for the "if" condition, but I have no idea to do the unit test if I don't introduce any new function since these variables are all private. Could you please give me some hint for adding a unit test to cover this part of code since I am new to swift developing. In my opinion, it is needless if we introduce some additional function in order to do the unit test. Thanks.

Reply:
I agree with you, we will solve coverage with a UI Test in the future that waits for the auto stop or something like that. Push the spaces changes and we can merge. 👌 Thanks for the effort! 🙌